Add the implementation of ConfigurationDiscovery on the OAP side. (#6220)
apache/skywalking: docs/en/setup/service-agent/java-agent/README.md

`plugin.kafka.topic_management` | Specify which Kafka topic name for the register or heartbeat data of Service Instance to report to. | `skywalking_managements` |
`plugin.springannotation.classname_match_regex` |  Match spring beans with regular expression for the class name. Multiple expressions could be separated by a comma. This only works when `Spring annotation plugin` has been activated. | `All the spring beans tagged with @Bean,@Service,@Dao, or @Repository.` |

## Dynamic Configurations

> All configurations above are static, if you need to change some agent settings in the runtime, please read [CDS - Configuration Discovery Service document](configuration-discovery.md) for more details.

Reviewer: ```suggestion
All configurations above are static, if you need to change some agent settings at runtime, please read [CDS - Configuration Discovery Service document](configuration-discovery.md) for more details.
```
